src/components/sporsmal/typer/tall-komp.tsx

```diff
@@ -21,7 +21,11 @@ const TallKomp = ({ sporsmal }: SpmProps) => {
 
     const onChange = (e: any) => {
         const value = e.target.value
-        setValue(sporsmal.id, value)
+        if (sporsmal.svartype == RSSvartype.BELOP) {
+            setValue(sporsmal.id, value / 100)
```

Comment:
Tror dette blir dobbelt opp

Reply:
🤔

Reply:
Har litt lyst å gå over å fjerne de fleste setValue, da de ikke trengs så lenge vi bruker sporsmal.id som name på inputet

Reply:
lurer litt på om jeg trengte dette når man går frem og tilbake. Skal teste litt uten igjen

Reply:
Stemmer at den var unødvendig @kulsveen . Ser du noen sideeffekter ved at jeg satt value={lokal} ? Eller er den kansje unødvendig også?

Reply:
De lokal statene syntes jeg klusser til ting, siden vi kan ende opp med å ha 2 forskjellige verdier for det samme spørsmålet. Men her tror jeg vi trenger den for å vite om underspørsmålet skal vises